test/solidity/oz/transparent-upgradeable-proxy/transparentProxy.js

```diff
@@ -0,0 +1,107 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const chai = require('chai')
+const { expect } = require('chai')
+const chaiAsPromised = require("chai-as-promised")
+const { ethers } = require('hardhat')
+const Constants = require('../../../constants')
+
+chai.use(chaiAsPromised);
+
+describe('@solidityequiv3 Transparent Upgradeable Proxy', function () {
+  let contractProxy, contractBox
+  let owner, signer, proxyAdminAddress
+  before(async function () {
+    [owner, signer] = await ethers.getSigners()
+    const factoryBox = await ethers.getContractFactory(Constants.Contract.Box)
+    contractBox = await factoryBox.deploy()
+    await contractBox.deployed()
+
+    const factory = await ethers.getContractFactory(Constants.Contract.MyCustomTransparentUpgradeableProxy)
+    contractProxy = await factory.deploy(contractBox.address, owner.address, [])
+    const receipt = await contractProxy.deployTransaction.wait()
+
+    proxyAdminAddress = receipt.events[2].args.newAdmin
+  })
+
+  it('should verify it calls the correct contract and method via proxy', async function () {
+    const signedTx = await owner.sendTransaction({
+        to: contractProxy.address,
+        data: '0x6057361d0000000000000000000000000000000000000000000000000000000000000008',
```

Comment:
Push those magic strings in a separate variables with proper names, so it's clear what they are.